Kickstarter-iOS/Views/Cells/DeprecatedRewardCell.swift

```diff
@@ -67,18 +67,18 @@ internal final class DeprecatedRewardCell: UITableViewCell, ValueCell {
     _ = self
       |> baseTableViewCellStyle()
       |> DeprecatedRewardCell.lens.accessibilityTraits .~ UIAccessibilityTraits.button.rawValue
-      |> (DeprecatedRewardCell.lens.contentView..UIView.lens.layoutMargins) %~~ { _, cell in
+      |> (DeprecatedRewardCell.lens.contentView .. UIView.lens.layoutMargins) %~~ { _, cell in
         cell.traitCollection.isRegularRegular
           ? .init(top: Styles.grid(2), left: Styles.grid(16), bottom: Styles.grid(4), right: Styles.grid(16))
           : .init(top: Styles.grid(1), left: Styles.grid(2), bottom: Styles.grid(2), right: Styles.grid(2))
       }
-      |> DeprecatedRewardCell.lens.contentView..UIView.lens.backgroundColor .~ projectCellBackgroundColor()
+      |> DeprecatedRewardCell.lens.contentView .. UIView.lens.backgroundColor .~ projectCellBackgroundColor()
       |> UIView.lens.contentMode .~ .top
 
     _ = self.rootStackView
       |> UIStackView.lens.spacing .~ Styles.grid(4)
       |> UIStackView.lens.layoutMargins
-        .~ .init(top: Styles.grid(3), left: Styles.grid(2), bottom: Styles.grid(2), right: Styles.grid(2))
+      .~ .init(top: Styles.grid(3), left: Styles.grid(2), bottom: Styles.grid(2), right: Styles.grid(2))
```

Comment:
This expression should probably start on previous line like so ... `..lens.layoutMargins .~ .init(top: ....)`